litmus-portal/frontend/src/schemas/index.ts

```diff
@@ -2,7 +2,7 @@ import { gql } from '@apollo/client';
 
 export const WORKFLOW_DETAILS = gql`
   query {
-    getWorkFlowRuns(project_id: "00000") {
+    getWorkFlowRuns(project_id: "00002") {
```

Comment:
No, you can register a cluster with any project ID while creating, mine was 00002, @amityt would raise a PR clanging the graphql schema to possible take a variable so that way the project ID would vary depending on the data they provided while creating the cluster.